vars/podman.groovy

```diff
@@ -0,0 +1,27 @@
+//Run podman remote build and push
+// Available parameters:
+//    cmds                     string -- Commands to be executed
```

Comment:
Instead of passing shell commands as a parameter, we could make this a custom step and let the user do whatever they want within. See e.g. how [`cosaPod`](https://github.com/coreos/coreos-ci-lib/blob/main/vars/cosaPod.groovy) works: the function is named `call` and it takes a `Closure` which we call using `body()`.

Reply:
Yep. That's what I was thinking.